commands/node.js

```diff
@@ -20,7 +21,16 @@ const runNode /*: Node */ = async ({
   args = [],
   stdio = 'inherit',
 }) => {
-  const params = ['-r', `${root}/.pnp.cjs`, ...getPassThroughArgs(args)];
+  const pnpESMLoaderPath = `${root}/.pnp.loader.mjs`;
+  const loaderArgs = existsSync(pnpESMLoaderPath)
```

Comment:
FWIW this is what yarn does https://github.com/yarnpkg/berry/blob/554257087edb4a103633e808253323fb9a21250d/packages/plugin-pnp/sources/index.ts#L31-L32 but it's a bit unfortunate that checking for `.pnp.loader.mjs` existence adds overhead